staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -367,6 +370,9 @@ func (c completedConfig) New(delegationTarget DelegationTarget) (*GenericAPIServ
 	if c.LoopbackClientConfig == nil {
 		return nil, fmt.Errorf("Genericapiserver.New() called with config.LoopbackClientConfig == nil")
 	}
+	if c.SharedInformerFactory == nil {
```

Comment:
I think we can tolerate this being nil to start.  It's a little weird, but I'm more interested in getting it wired than chasing an initialization flow through federation unit tests.